docs/source/example_use.rst

```diff
@@ -1,3 +1,343 @@
+<https://github.com/unlimitedlabs/orchestra/tree/stable/journalism_workflow>`_.
+
+************
+The workflow
+************
+.. image:: ../static/img/journalism_workflow/workflow.jpg
+   :height: 300px
+
+The image above depicts our example workflow, which is composed of the
+following steps:
+
+* An editor finds a good story and sends a reporter off to investigate.
+
+* The reporter writes up a draft article.
+
+* A more experienced reporter then reviews the article and suggests
+  improvements.
+
+* In parallel with the reporting step, a photographer captures photos for the
+  story.
+
+* A senior photographer reviews the photos and selects the best ones.
+
+* The selected photos are resized and recolored for display across different
+  media.
+
+* Finally, a copy editor adds headlines and photo captions to complete the
+  story.
+
+To make things work in practice, there's also a hidden machine step at the
+beginning of the workflow to set up some google documents and folders for
+article writing and image storage.
+
+********************
+Running the workflow
+********************
+
+Setup
+=====
+
+If you haven't followed the :doc:`getting started guide <getting_started>` to
+set up Orchestra yet, you should do that now. Also, make sure that
+``'journalism_workflow'`` is in your ``INSTALLED_APPS`` setting.
+
+Additionally, the journalism workflow requires Google Apps integration to run,
+so make sure in ``orchestra_settings.py`` you set ``settings.GOOGLE_APPS`` to
+``True``, and fill in values for ``settings.GOOGLE_SERVICE_EMAIL``,
+``settings.GOOGLE_P12_PATH``, and ``settings.GOOGLE_PROJECT_ROOT_ID``.
+
+Next, make sure you have the journalism workflow fixtures installed by running
+(if you haven't already) ``python manage.py loaddata journalism_workflow``. This
+will create the following accounts:
+
+* username: ``journalism-editor``, password: ``editor``. A worker with
+  ``editor`` certification.
+
+* username: ``journalism-reporter-1``, password: ``reporter``. A worker
+  with entry-level ``reporter`` certification.
+
+* username: ``journalism-reporter-2``, password: ``reporter``. A worker
+  with review-level ``reporter`` certification.
+
+* username: ``journalism-photographer-1``, password: ``photographer``. A
+  worker with entry-level ``photographer`` certification.
+
+* username: ``journalism-photographer-2``, password: ``photographer``. A
+  worker with review-level ``photographer`` certification.
+
+* username: ``journalism-copy-editor``, password: ``copy-editor``. A worker
+  with ``copy_editor`` certification.
+
+Finally, we've included a management script to start and monitor the workflow.
+Download it to the directory of your project next to ``manage.py`` with::
+
+  wget https://raw.githubusercontent.com/unlimitedlabs/orchestra/stable/example_project/journalism_workflow_ctl.py
+
+Start the workflow
+=====================
+
+To start the workflow, make sure Orchestra is running (``python manage.py
```

Comment:
initially i read past this.  maybe make two bullet points: 1) runserver, 2) --new